definitions/de.yaml

```diff
@@ -11,18 +19,57 @@ months:
   - name: Karfreitag
     regions: [de]
     function: easter(year)-2
+  - name: Ostersonntag
+    regions: [de]
+    function: easter(year)
+    type: informal
   - name: Ostermontag
     regions: [de]
     function: easter(year)+1
   - name: Christi Himmelfahrt
     regions: [de]
     function: easter(year)+39
+  - name: Pfingstsonntag
+    regions: [de]
+    function: easter(year)+49
+    type: informal
   - name: Pfingstmontag
     regions: [de]
     function: easter(year)+50
   - name: Fronleichnam
     regions: [de_bw, de_by, de_he, de_nw, de_rp, de_sl]
     function: easter(year)+60
+  - name: Fronleichnam
+    regions: [de_sn_aux, de_th_aux]
+    function: easter(year)+60
+  - name: Weiberfastnacht
+    regions: [de_bw, de_by, de_he, de_nw, de_rp, de_sl, de_sn ]
+    function: easter(year)-52
+    type: informal
+  #~ - name: Fastnachtfreitag
```

Comment:
Hello,
the commented definitions are "very informal" holidays, you can remove them,